public-ui/src/app/data-browser/charts/chart-measurement-values/chart-value-react.component.tsx

```diff
@@ -133,6 +161,117 @@ export class ValueReactChartComponent extends React.Component<Props, State> {
+                tooltipText = tooltipText + ' &ge; <b>' + a.stratum4.replace('>=', '')
+                    + '</b> <br/>' + participantCountText + '</div>';
+            } else {
+                tooltipText = tooltipText + ' <b>' + a.stratum4
+                    + '</b> <br/>' + participantCountText + '</div>';
+            }
+        } else {
+            tooltipText = '<div class="chart-tooltip"> <b>' + analysisStratumName + '</b>' +
+                '<br/>' + 'Measurement Value : <b>' + a.stratum4
+                + '</b> <br/>' + participantCountText + '</div>';
+        }
+        data.push({
+                name: a.stratum4, y: a.countValue, thisCtrl: this,
+                result: a, toolTipHelpText: tooltipText, binWidth: a.stratum6,
+                analysisId: a.analysisId
+        });
+    }
+    const lessThanData = data.filter(
+          d => d.name != null && d.name.indexOf('< ') >= 0);
+    const greaterThanData = data.filter(
+          d => d.name != null && d.name.indexOf('>= ') >= 0);
+    data = data.filter(
+           d => d.name != null && (d.name.indexOf('< ') === -1 && d.name.indexOf('>= ') === -1));
+    data.sort((a, b) => {
+          let aVal: any = a.name;
+          let bVal: any = b.name;
+          // Sort  numeric data as number
+          if (a.name.indexOf(' - ') > 0) {
+            aVal = a.name.split(' - ')[1];
+          } else if (a.name.indexOf('< ') >= 0) {
+            aVal = a.name.replace('< ', '');
+          } else if (a.name.indexOf('>= ') >= 0) {
+            aVal = a.name.replace('>= ', '');
+          }
+          if (b.name.indexOf(' - ') > 0) {
+            bVal = b.name.split(' - ')[1];
+          } else if (b.name.indexOf('< ') >= 0) {
+            bVal = b.name.replace('< ', '');
+          } else if (b.name.indexOf('>= ') >= 0) {
+            bVal = b.name.replace('>= ', '');
+          }
+          if (isNaN(Number(aVal))) {
+            // Don't do anything
+          } else {
+            // Make a number so sort works
+            aVal = Number(aVal);
+            bVal = Number(bVal);
+          }
+          return bVal - aVal;
+        });
+    if (greaterThanData.length > 0) {
+        data.unshift(greaterThanData[0]);
+    }
+    if (lessThanData.length > 0) {
+        data.push(lessThanData[0]);
+    }
+    if (data.length > 2) {
+      if (greaterThanData.length === 0) {
+        if (isNaN(Number(data[0].name))) {
+          // Don't do anything
+        } else {
+          data[0].name = '>= ' + data[0].name;
+        }
+      }
+      if (lessThanData.length === 0) {
+        if (isNaN(Number(data[data.length - 1].name))) {
+          // Don't do anything
+        } else {
+          data[data.length - 1].name = '< ' + data[data.length - 1].name;
+        }
+      }
+    }
+    for (const d of data) {
+        cats.push(d.name);
+    }
+    const temp = data.filter(x => x.y > 20);
+    const dataOnlyLT20 = temp.length > 0 ? false : true;
```

Comment:
Nit: can be simplified
```suggestion
    const dataOnlyLT20 = temp.length <= 0;
```